utils/start_time.py

```diff
@@ -0,0 +1,45 @@
+from pathlib import Path
+import os
+import time
+import datetime
+import json
+
+async def save_start_time():
```

Comment:
I think this doesn't need to be `async`, since it doesn't use `await`.

(It's also not called with `await` which may be a problem, but again it doesn't need to be `async` in the first place.)

Reply:
> (It's also not called with await which may be a problem, but again it doesn't need to be async in the first place.)

That part was wrong, I was looking at `get_uptime`'s invocation instead. In any case tho, the fix in https://github.com/Splatoon-Stronghold/cas-stronghold-bot/pull/13/commits/f298fe3d33c0f79acd0c9a1bec39ef53b9eb51f3 is good.